server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -169,6 +173,32 @@ public static IndexRequest getIndexWriteRequest(DocWriteRequest<?> docWriteReque
         return indexRequest;
     }
 
+    public static BulkRequest wrapSingleRequest(final ReplicatedWriteRequest<?> request) {
```

Comment:
Also good with moving it to `BulkRequest` -- leaving it here to keep it together with `toSingleResponse`